public/events/mission.html

```diff
@@ -731,12 +734,12 @@ <h4 class="panel-title">
         return;
       }
       vm.isSubmitting=true;
+
       var body={
         status: vm.mission.event_status === 'complete' ? 'complete' : 'active',
         metadata: vm.mission,
-        eventId: currentEventId
+        eventId: currentEventId ? currentEventId : null
```

Comment:
@matthewberryman @qclin   I guess the ID of events and mission tables should be unique and identical ? If so, better to use `currentMissionId` everywhere when talking to missions endpoint.  `currentEventId` is a global var? and I do not see it being defined in the scope of this modal ?